Stronger/Info/ExerciseWeek.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ExerciseWeek.swift
+//  Stronger
+//
+//  Created by Theodore Kanell on 2/19/24.
+//
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct ExerciseWeek: View {
+    var value: String
+    var body: some View {
+        HStack {
+            NavigationLink(destination: WorkoutHome(presentingAccount: .constant(false))) {
```

Comment:
The binding here should not be constant, it will result in the fact that you will never be able to show the account sheet in there. Might not even be needed in the subviews as long as the account button is shown in the views at the root level of the tab bar.